earthaccess/api.py

```diff
@@ -283,6 +283,7 @@ def download(
     provider: Optional[str] = None,
     threads: int = 8,
     *,
+    hide_progress: bool = False,
```

Comment:
I don't feel we should add a new kwarg here. Currently `pqdm_kwargs.disable` allows the user to control this behavior, and perhaps this PR should just update the default value instead of introducing a new API?

Reply:
I.e.

```python
pqdm_kwargs = {
    "n_jobs": threads,
    "disable": not _is_interactive(),
    **(pqdm_kwargs or {}),
}
```

Reply:
That's definitely true. You've kinda convinced me :) this is end-user-facing and we want it to be user-friendly. Maybe it should be `show_progress` instead? `hide_progress=False` is a double-negative.